class-tumblr-import.php

```diff
@@ -482,8 +604,15 @@ public function do_posts_import( $url ) {
 
 					$post['post_author'] = $this->blog[ $url ]['post_author'];
 
+					do_action( 'tumblr_importer_importing_post_before', $post );
+
 					do_action( 'tumblr_importing_post', $post );
+
+					do_action( 'tumblr_importer_importing_post_after', $post );
+
+					do_action( 'tumblr_importer_insert_new_post_before', $post );
 					$id = wp_insert_post( $post );
```

Comment:
Thoughts on turning the third parameter of this function `false`?

https://developer.wordpress.org/reference/functions/wp_insert_post/

Might help avoid things like social plugins firing off 500 updates as posts are inserted.

Reply:
I think that's a good idea. Updated both places.